ext/vscode/ci-package.ps1

```diff
@@ -0,0 +1,25 @@
+param(
+    $PackageArguments
+)
+
+# Define OS targets and corresponding RIDs
+$osList = @(
+    @{ Name = "win-x64"; Rid = "win-x64" },
```

Comment:
Azure MCP already ships win arm64 and osx arm64. Is there a reason to not inlucde them?